src/bindgen/parser.rs

```diff
@@ -98,8 +96,10 @@ struct Parser<'a> {
 }
 
 impl<'a> Parser<'a> {
-    fn should_parse_dependency(&self, pkg_name: &String) -> bool {
-        if self.parsed_crates.contains(pkg_name) {
+    fn should_parse_dependency(&self, pkg_name: &str) -> bool {
+        let pkg_name = pkg_name.to_string();
```

Comment:
This allocates for no good reason, while before it didn't. I agree that `&String` is generally useless, but this is worse, so let's annotate it with `#[allow(..)]` instead, and maybe change the `Vec::contains` calls in a follow-up to `.iter().any()` or something else that allows us to use a `&str`.